gsuite/client.py

```diff
@@ -48,4 +47,55 @@ def list_users(self):
             return []
 
         resp = self.directory_client.users().list(domain=self.domain).execute()
-        return resp["users"]
+        users = resp.get("users", [])
+        if resp.get("nextPageToken", ""):
+            while True:
+                pageToken = resp["nextPageToken"]
+                resp = (
+                    self.directory_client.users()
+                    .list(domain=self.domain, pageToken=pageToken)
+                    .execute()
+                )
+                users += resp["users"]
+                if not resp.get("nextPageToken", ""):
+                    break
+        return users
+
+    def list_groups(self):
+        if self.offline:
+            return []
+
+        resp = self.directory_client.groups().list(domain=self.domain).execute()
+        groups = resp.get("groups", [])
+        if resp.get("nextPageToken", ""):
+            while True:
+                pageToken = resp["nextPageToken"]
+                resp = (
+                    self.directory_client.groups()
+                    .list(domain=self.domain, pageToken=pageToken)
+                    .execute()
+                )
+                groups += resp["groups"]
+                if not resp.get("nextPageToken", ""):
+                    break
+        return groups
+
+    def list_members_of_group(self, group):
+        if self.offline:
+            return []
+
+        resp = self.directory_client.members().list(groupKey=group).execute()
+        members = resp.get("members", [])
+        if resp.get("nextPageToken", ""):
+            while True:
+                pageToken = resp["nextPageToken"]
+                resp = (
+                    self.directory_client.members()
+                    .list(groupKey=group, pageToken=pageToken)
+                    .execute()
+                )
+                members += resp["members"]
+                if not resp.get("nextPageToken", ""):
+                    break
```

Comment:
Surprised the goog api client doesn't do this. Should make an issue for de-duping this pagination logic.

Reply:
My bad. The documentation for this library is super scattered and I missed that it does have pagination helpers. Pushed up a fix

Reply:
:+1: no worries